compiler/AST/ForLoop.cpp

```diff
@@ -62,9 +63,9 @@
  * - "for in in (1..)"             // doesn't handle unbounded ranges
  * - "for i in 1..10 by 2 by 2"    // doesn't handle more than one by operator
  * - "for i in 1..10 align 2"      // doesn't handle align operator
- * - "for i in 1..#10"             // doesn't handle # operator
+ * - "for i in 1..#10 by 2"        // doesn't handle strided and counted ranges
```

Comment:
Totally optional not to mention it, but it also won't handle cases like:

    for i in (1..10)#3